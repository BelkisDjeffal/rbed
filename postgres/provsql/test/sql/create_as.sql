\set ECHO none
SET search_path TO provsql_test, provsql;

CREATE TABLE x (n integer);
INSERT INTO X VALUES (10),(20);
SELECT add_provenance('x');

CREATE TABLE u AS (SELECT * FROM x);

SELECT attname, typname
FROM pg_attribute JOIN pg_type ON atttypid=oid
WHERE attrelid ='u'::regclass AND attnum>0
ORDER BY attname;

DROP TABLE x;
DROP TABLE u;
