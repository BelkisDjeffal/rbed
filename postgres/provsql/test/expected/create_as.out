\set ECHO none
 add_provenance 
----------------
 
(1 row)

 attname | typname 
---------+---------
 n       | int4
 provsql | uuid
(2 rows)

